op_crates/websocket/lib.rs

```diff
@@ -311,49 +308,25 @@ pub async fn op_ws_next_event(
   let cancel = RcRef::map(resource, |r| &r.cancel);
   let val = rx.next().or_cancel(cancel).await?;
   let res = match val {
-    Some(Ok(Message::Text(text))) => NextEventResponse {
-      kind: "string".to_string(),
-      data: Some(NextEventResponseData::String(text)),
-    },
+    Some(Ok(Message::Text(text))) => NextEventResponse::String(text),
     Some(Ok(Message::Binary(data))) => {
       // TODO(ry): don't use json to send binary data.
```

Comment:
Is the TODO still valid or is the `NextEventResponse::Binary(data)` now fully binary instead of JSON?

Reply:
As @AaronO mentioned, this still is an issue. It will be soon worked on.